projections/account_test.go

```diff
@@ -0,0 +1,92 @@
+package projections_test
+
+import (
+	"testing"
+
+	"github.com/dogmatiq/example/internal/database"
+	"github.com/dogmatiq/example/internal/testrunner"
+	"github.com/dogmatiq/example/messages/commands"
+	"github.com/dogmatiq/testkit"
+	"github.com/dogmatiq/testkit/engine"
+)
+
+func Test_AccountProjectionHandler(t *testing.T) {
+	t.Run(
+		"when an account is opened for a new customer",
```

Comment:
What about when an account is opened for an existing customer?

Reply:
Good point.